app/date-comment-helpers.js

```diff
@@ -34,50 +34,43 @@ class DateCommentHelpers {
     if (correctComment) {
       const commentDateMatch = myRegex.exec(correctComment.data.text);
       const commentDate = commentDateMatch[1];
-      const commMoment = moment(commentDate, 'MM/DD/YYYY').toISOString();
-      return commMoment;
+      const isoDate = moment(commentDate, 'MM/DD/YYYY').toISOString();
+      return isoDate;
     }
     return false;
   }
 
-  findPrevMoveDateFromComments(opts) {
-    const actionList = opts.actionList;
-    const cardCreationDate = opts.cardCreationDate;
-    const commentList = opts.commentList;
-    let correctComment = false;
-    if (commentList) {
-      correctComment = this.checkCommentsForDates(commentList, true, false);
-    }
-    if (correctComment) {
-      return correctComment;
-    } else if (actionList) {
-      if (actionList.length > 1) {
-        return actionList[0].date;
-      }
+  getNewCommentFromDate(deletedNewComment, comments, altToDate) {
+    if (altToDate) {
```

Comment:
One thing this function highlights for me is that I don't know what all of these functions are supposed to do.  As a separate PR, it might be useful if we swarmed on adding short one/two-line comments at the top of some of the functions.